pkg/nimble/netif/nimble_netif.c

```diff
@@ -0,0 +1,630 @@
+/*
+ * Copyright (C) 2018-2019 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     pkg_nimble_netif
+ * @{
+ *
+ * @file
+ * @brief       GNRC netif wrapper for NimBLE
+ *
+ * @author      Hauke Petersen <hauke.petersen@fu-berlin.de>
+ *
+ * @}
+ */
+
+#include <limits.h>
+#include <errno.h>
+
+#include "assert.h"
+#include "thread.h"
+#include "thread_flags.h"
+
+#include "net/ble.h"
+#include "net/bluetil/addr.h"
+#include "net/gnrc/netif.h"
+#include "net/gnrc/netif/hdr.h"
+#include "net/gnrc/netreg.h"
+#include "net/gnrc/pktbuf.h"
+#include "net/gnrc/nettype.h"
+
+#include "nimble_netif.h"
+#include "nimble_netif_conn.h"
+#include "nimble_riot.h"
+#include "host/ble_gap.h"
+#include "host/util/util.h"
+
+#define ENABLE_DEBUG            (0)
+#include "debug.h"
+
+#ifdef MODULE_GNRC_SIXLOWPAN
+#define NETTYPE                 GNRC_NETTYPE_SIXLOWPAN
+#elif defined(MODULE_GNRC_IPV6)
+#define NETTYPE                 GNRC_NETTYPE_IPV6
+#else
+#define NETTYPE                 GNRC_NETTYPE_UNDEF
+#endif
+
+/* buffer configuration
+ * - we need one RX and one TX buffer per connection */
+#define MTU_SIZE                (NIMBLE_NETIF_MTU)
```

Comment:
Not a blocking comment, but why is `MTU_SIZE` needed?

Reply:
For calculation of the needed buffer sizes. But if your question is, why not simply to use NIMBLE_NETIF_MTU, then it is for historical reason and to make some of the lines easier to read (line length issues)...

Reply:
Ok.